packages/gestalt/src/ExpandableModule.flowtest.js

```diff
@@ -0,0 +1,29 @@
+// @flow strict
+import React from 'react';
+import ExpandableModule from './ExpandableModule.js';
+
+const Valid = (
+  <ExpandableModule
```

Comment:
@galaxy0101 this should now break CI, since there is no `id`, `accessibilityExpandLabel` or `accessibilityCollapseLabel` defined on the root component

```
type Props = {|
  id: string,
  accessibilityExpandLabel: string,
  accessibilityCollapseLabel: string,
```

Reply:
aha, forgot to run flow test.